src/printer.js

```diff
@@ -536,7 +536,7 @@ function genericPrintNoParens(path, options, print, args) {
         return printStatementSequence(bodyPath, options, print);
       }, "body");
 
-      const hasContent = getFirstString(naked);
+      const hasContent = n.body.find(node => node.type !== "EmptyStatement");
```

Comment:
Note that we skip EmptyStatements from being listed as valid targets for attaching comments, so we don't need to ensure that there's no attached comments.